sms_ovh_http/README.rst

```diff
@@ -38,10 +38,22 @@ Configuration
 
 To configure this module, you need to:
 
-* Go to IAP menu
+* Go to settings > technical > IAP Account
 * Create a new account with **SMS OVH HTTP** as provider
-* Fill your account information
-* You can now send an sms
+* Fill your account information as follows:
+    * SMS Account
+        .. image:: https://raw.githubusercontent.com/OCA/connector-telephony/14.0/sms_ovh_http/sms_ovh_http/static/img/img.png
```

Comment:
This entry is not correct.

**Example:**
`.. figure:: ../static/description/location-tray.png`

Next entries are not correct too.